lib/api/cln/cln_client.dart

```diff
@@ -73,12 +73,9 @@ class CLNApi extends AppApi {
     return client.call<CLNListFundsRequest, AppListFunds?>(
         method: "listfunds",
         params: params,
-        onDecode: (jsonResponse) {
-          LogManager.getInstance.debug(
-              "Grpc list funds call ${AppListFunds.fromJSON(jsonResponse as Map<String, dynamic>, snackCase: mode == ClientMode.unixSocket)}");
-          return AppListFunds.fromJSON(jsonResponse,
-              snackCase: mode == ClientMode.unixSocket);
-        });
+        onDecode: (jsonResponse) => AppListFunds.fromJSON(
+            jsonResponse as Map<String, dynamic>,
+            snackCase: mode == ClientMode.unixSocket));
```

Comment:
we can keep this? have print debug in this phase helps a lot to catch bugs

Reply:
ok, I removed print debug because it does not make sense and the error will be shown in debug statement instead the actual place. because the error is in `AppListFunds.fromJSON` !!

Reply:
yes but we see the input that cause this error, from the issue that I opened we are able to see that the jsonResponse is a empty one

Reply:
okay will make the debug statement with jsonResponse